terraform/variables.tf

```diff
@@ -59,3 +59,10 @@ variable "endpoint_bindings" {
   type        = map(string)
   default     = {}
 }
+
+# additional variables
+variable "tls" {
+  description = "Whether TLS should be enabled"
+  type        = bool
+  default     = false
+}
```

Comment:
newb opensearch question: isn't it a requirement that opensearch have TLS enabled? Is that also true for its dashboards?

Reply:
OpenSearch yes, opensearch-dashboards (the current) however, no.